To avoid dependency on libtool/libltdl, change "*.la" extension to "*.so".
This patch comes from in-house.  It has not been submitted upstream; plans
unclear.

--- ImageMagick-6.8.3/magick/module.c.~1~	2012-12-07 05:14:16.000000000 -0800
+++ ImageMagick-6.8.3/magick/module.c	2015-02-10 20:31:54.595614457 -0800
@@ -72,7 +72,7 @@
   Define declarations.
 */
 #if defined(MAGICKCORE_LTDL_DELEGATE)
-#  define ModuleGlobExpression "*.la"
+#  define ModuleGlobExpression "*.so"
 #else
 #  if defined(_DEBUG)
 #    define ModuleGlobExpression "IM_MOD_DB_*.dll"
@@ -1453,7 +1453,7 @@
   (void) LogMagickEvent(TraceEvent,GetMagickModule(),"%s",tag);
   assert(name != (char *) NULL);
 #if defined(MAGICKCORE_LTDL_DELEGATE)
-  (void) FormatLocaleString(name,MaxTextExtent,"%s.la",tag);
+  (void) FormatLocaleString(name,MaxTextExtent,"%s.so",tag);
   (void) LocaleLower(name);
 #else
 #if defined(MAGICKCORE_WINDOWS_SUPPORT)
@@ -1504,7 +1504,7 @@
 #if !defined(MAGICKCORE_LTDL_DELEGATE)
   (void) FormatLocaleString(name,MaxTextExtent,"%s.dll",tag);
 #else
-  (void) FormatLocaleString(name,MaxTextExtent,"%s.la",tag);
+  (void) FormatLocaleString(name,MaxTextExtent,"%s.so",tag);
 #endif
 }
 
